src/Components/AddTask.jsx

```diff
@@ -0,0 +1,46 @@
+import React , { useState} from "react";
+import Button from "./Button";
+
+function AddTask({ addTask }) {
+    const [taskInput, setTaskInput] = useState("");
+    const [status, setStatus] = useState("Todo");
+
+    const taskStatuses = ["Todo", "In-Progress", "Completed"];
+    
+    const createClick = () => {
+        if(taskInput.trim()) {
+            addTask({task: taskInput, status});
+            setTaskInput(""); // clear after
+            setStatus("Todo");
+        }
+    }
+    return (
+        <div className="todo-input">
+            <input 
+                type="text" 
+                placeholder="Enter todo" 
+                value={taskInput} onChange={(e) => setTaskInput(e.target.value)} maxLength={25}>
```

Comment:
We can trim the textValue from here so that we can avoid using trim() everywhere.

`setTaskInput(e.target.value.trim())`